docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -168,38 +172,54 @@ Grafana supports user authentication through Okta, which is useful when you want
+| `assertion_attribute_role`                                 | No       | Friendly name or name of the attribute within the SAML assertion to use as the user roles                                                                                                                    |                                                       |
+| `assertion_attribute_org`                                  | No       | Friendly name or name of the attribute within the SAML assertion to use as the user organization                                                                                                             |                                                       |
+| `allowed_organizations`                                    | No       | List of comma- or space-separated organizations. User should be a member of at least one organization to log in.                                                                                             |                                                       |
+| `org_mapping`                                              | No       | List of comma- or space-separated Organization:OrgId:Role mappings. Organization can be `*` meaning "All users". Role is optional and can have the following values: `Viewer`, `Editor` or `Admin`.          |                                                       |
+| `role_values_editor`                                       | No       | List of comma- or space-separated roles which will be mapped into the Editor role                                                                                                                            |                                                       |
+| `role_values_admin`                                        | No       | List of comma- or space-separated roles which will be mapped into the Admin role                                                                                                                             |                                                       |
+| `role_values_grafana_admin`                                | No       | List of comma- or space-separated roles which will be mapped into the Grafana Admin (Super Admin) role                                                                                                       |                                                       |
+| `name_id_format`                                           | No       | Name ID Format to request within the SAML assertion                                                                                                                                                          | `urn:oasis:names:tc:SAML:2.0:nameid-format:transient` |
```

Comment:
This isn't clear to me, not sure what this means.

Reply:
It's a table with the name of the config, if it's required, technical description and default value

This table seems to serve more as a quick reference guide than long explanation

Reply:
I agree.